docs/README.adoc

```diff
@@ -137,9 +137,9 @@ the LinuxCNC community.
 
 === Drawings from QCAD
 
-- Save as .dxf to the same directory as the image
-- Set line thickness to 0.50mm
-- Export the  image as 480 x 480 Resolution Auto .png
+- Save as .dxf to the same directory as the image.
+- Set line thickness to 0.50 mm.
```

Comment:
What about the thin spaces? Or do we want to insert them all with one commit?

Reply:
I had not thought about the thin spaces. To have a dedicated concerted action on all the units sounds good.